packages/lesswrong/lib/collections/posts/constants.ts

```diff
@@ -31,3 +35,16 @@ export const postStatusLabels = [
     label: 'deleted'
   }
 ];
+
+const amaTagIdSetting = new DatabasePublicSetting<string | null>('amaTagId', null)
+const openThreadTagIdSetting = new DatabasePublicSetting<string | null>('openThreadTagId', null)
+const startHerePostIdSetting = new DatabasePublicSetting<string | null>('startHerePostId', null)
```

Comment:
I like the approach which is like, "we either know it's a special post by it's category (tag), or we can declare it's a special post directly by it's ID. Maybe at some point we'll want to do some sort of 'all tags written by an official account' or something, but we'll cross that bridge if we come to it."

However, now that you've done this, it seems suuuper tempting to make a database setting that's like:

```
  iconTags: [['asl89a', 'QuestionAnswerIcon'], ['a83456as', 'AllInclusiveIcon']],
  iconPosts: [['sdf6kt', 'PlayCircleOutlineIcon']],
```

The hard part will be getting those strings into actual icons. One thing you _can't_ do, is to ship all the MUI icons to the frontend. If you naively try to import all the MUI icons that's what will happen. I claim that a) we're not doing that now, and b) that will significantly impact performance. But I could be wrong about either of those.

All the above makes it a pretty hard problem. But if you could do it, it'd be super flexible, and I'd say it's worth some hours of investigation.

Reply:
Hmm, yeah that's interesting idea. I looked into this for a bit and so far haven't thought of a viable implementation. Also a concern I have with this strategy is that it binds us to Material UI icons; custom icons (such as the icon we used for pinned posts) and icons from other libraries wouldn't work with this, unless I'm underthinking it.